parsec/_parsec_pyi/backend_connection.pyi

```diff
@@ -0,0 +1,10 @@
+from oxidation.bindings.generator.api import DeviceID
+from parsec._parsec_pyi.addrs import BackendOrganizationAddr
+from parsec._parsec_pyi.crypto import SigningKey
+from parsec._parsec_pyi.protocol.ping import AuthenticatedPingRep
+
+class AuthenticatedCmds:
+    def __init__(
+        self, server_url: BackendOrganizationAddr, device_id: DeviceID, signing_key: SigningKey
+    ) -> None: ...
+    async def ping(self, ping: str) -> AuthenticatedPingRep: ...
```

Comment:
Seems good, but in a near future there would be no `ping` method (or any other cmd) but just a send 

```python
    async def send(self, ReqType) -> Type[ReqType.Response]
```

Reply:
I added this method to get a `POC` asap.

Reply:
Yes I understood

Reply:
Do you think we can merge it as it is or we should already adopt this pattern?

Reply:
It doesn't affect the actual app so it won't cause problem in production
We could merge it and iterate over it